react/features/chat/actionTypes.js

```diff
@@ -13,6 +13,16 @@
  */
 export const ADD_MESSAGE = Symbol('ADD_MESSAGE');
 
+/**
+ * The type of the action which signals to clear messages in chat on
+ * connecting to conference.
+ *
+ * {
+ *     type: CLEAR_CHAT
+ * }
+ */
+export const CLEAR_CHAT = Symbol('CLEAR_CHAT');
```

Comment:
Since we have ADD message and SEND message, I think it feels a bit more consistent to thave some CLEAR_MESSAGES action here.